### ISSUE TITLE
Accessibility on the WebView window

### ISSUE THREAD

**Issue by hallivia** [OutSystems/WebView#329]
Hi there,

I seem to have successfully triggered the webview window, but using TAB key on the keyboard, I cannot access/input to the textbox. 

What is the recommended way to do this?

Thank you

**Comment by joaompneves** [comment]
Hi,

Can you please explain better what exactly are you trying to accomplish?

**Comment by hallivia** [comment]
Hi, yes. So we want to make sure our Avalonia App is accessible, meaning the texts can be read using a screen reader. We noticed when the window pops up, using the tab key on the keyboard, we cannot get into the window's components, like the text in the window. How can we allow that to happen? Basically the tab key only allows navigating through the window's header.

Thank you

**Comment by joaompneves** [comment]
Well ... i don't quite understand how the tab navigation works in Avalonia. When I try to set set a control with IsTabStop=true, the control never gets focus. I'm missing some Avalonia context regarding Tab navigation, to be able to help.

**Comment by joaompneves** [comment]
Actually found the issue. WebView was missing `Focusable=True"

**Comment by hallivia** [comment]
Hi there,

adding the focus helps make the window focusable, but we still can't focus on the textblocks or text boxes in the window for the screen reader to read it. Any other suggestion? 
tabbing values using the tab key seems to work when using normal browsers, so hoping we can get this to work as well to support accessibility for our customers.

Thank you

**Comment by hallivia** [comment]
![sign in focus](https://github.com/OutSystems/WebView/assets/148501207/aefd8502-6a7b-41aa-b7cb-683f4702f5af)
the focus is only on the window, and cannot be on the actual components inside the window like the "Sign In" text, which is supported on normal browser

**Comment by joaompneves** [comment]
Hum, it works with google, as you can see:



https://github.com/OutSystems/WebView/assets/693294/3459dc34-5f2c-40c4-9ca6-8afb9673995d

**Comment by hallivia** [comment]
Hi  there,

I noticed you are on macOS, and I just tried today with Focusable=True and I can't even tab into the WIndow. Is there any other change you made to make it work?
I tried with google.com as well

Thank you

**Comment by hallivia** [comment]
I just tried on SampleWebView project (example project) you had and it works. I will look into what is different on my project that is causing the issue. Thank you

**Comment by hallivia** [comment]
I looked into this for some time today and tried with IsTabStop, Focusable, and TabIndex and can't seem to make it work when using Avalonia 11. Would you possibly have any thoughts as to how to workaround this?

**Comment by joaompneves** [comment]
Made one commit [here](https://github.com/OutSystems/WebView/pull/310). Please check it out now.

**Comment by hallivia** [comment]
Hi Joao, thanks for the deep dive here. I looked at the commit and made the changes on my side following https://github.com/OutSystems/WebView/pull/310/commits/3d876c8b4d2746b1f886efaabdda871b68368e9d. I am however still only able to tab through the header and can only tab on the content once I click on it. 

Showing video

https://github.com/OutSystems/WebView/assets/148501207/09fdcf9e-09fe-478c-86d6-e4d29ad306fd

**Comment by joaompneves** [comment]
Can you check out the [branch](https://github.com/OutSystems/WebView/tree/avalonia11) and try?

I dont have a windows machine, so I can't help with OS specific issues.

**Comment by hallivia** [comment]
Hi Joao. I tried using the branch. I had to add 

.With(new Win32PlatformOptions { CompositionMode=  new[]
                      {
                          Win32CompositionMode.RedirectionSurface
                      } })

to make it work on Windows. However, it does work. The problem is when I replaced the WebViewControl.Avalonia with the one from Nuget (<PackageVersion Include="WebViewControl-Avalonia" Version="2.117.3-avalonia11" />), navigating doesn't seem to work. I still haven't found the difference yet.

**Comment by joaompneves** [comment]
The one from nuget is not updated.

**Comment by joaompneves** [comment]
Just released [2.117.4-avalonia11](https://www.nuget.org/packages/WebViewControl-Avalonia/2.117.4-avalonia11)
Check it out

**Comment by hallivia** [comment]
Joao,

thank you so much! the new version seems to work. One thing I notice that I'd like to get your opinion:
On Windows, upon tabbing through the component in the webview like the button on google.com, the Narrator app is able to narrate each component
On Mac, VoiceOver isn't able to narrate it. 

Any idea if this is an improvement that can be made on your side for the ARM64 version? Or is this something that needs to be brought up to VoiceOver? 

Thank you

**Comment by joaompneves** [comment]
Released a new version that introduces a new webview method: `EnableAccessibility`. Check it out.

**Comment by hallivia** [comment]
Hi Joao,

Tested the new version and narration works on both platforms. I think I have everything now. I just want to say thank you so much for the fast response and help on this. I really appreciate it. I'm glad to be using this WebView library.
